### PR TITLE
Minor localization fixes

### PR DESCRIPTION
**Pull request by simfeld**

### CODE REVIEW

File: common/locales/de.json [hunk 1]
```diff
@@ -100,7 +100,6 @@
     },
     "category": {
       "fields": {
-        "activityType": "Typ",
```
Comment (simfeld) [inline common/locales/de.json:103]: Not used anywhere

File: frontend/src/locales/en.json [hunk 2]
```diff
@@ -46,7 +46,6 @@
       },
       "campDangerzone": {
         "deleteCamp": {
-          "action": "Delete",
```
Comment (simfeld) [inline frontend/src/locales/en.json:49]: Uses `global.button.delete` now

File: frontend/src/locales/en.json [hunk 3]
```diff
@@ -136,13 +135,12 @@
...
       "backToHome": "Back to home",
       "error": "An unexpected error occurred",
-      "login": "Login",
       "notFound": "This invitation is already accepted or was deleted",
       "openCamp": "Open camp",
       "register": "Register",
       "reject": "Deny invitation",
       "successfullyRejected": "The invitation was successfully rejected",
-      "title": "Invitation to camp",
+      "title": "Invitation to camp \"{campName}\"",
```
Comment (simfeld) [inline frontend/src/locales/en.json:146]: Interpolate camp name

File: frontend/src/locales/en.json [hunk 4]
```diff
@@ -237,6 +235,7 @@
       "deleteEntity": "Delete {entity}",
       "edit": "Edit",
       "editable": "Editable",
+      "login": "Log in",
```
Comment (simfeld) [inline frontend/src/locales/en.json:258]: Translate login only once

File: frontend/src/locales/en.json [hunk 5]
```diff
@@ -413,9 +406,6 @@
         "previewIframeTitle": "Print preview",
         "title": "Print preview"
       },
-      "sideBarPeriods": {
-        "title": "Periods"
-      },
```
Comment (simfeld) [inline frontend/src/locales/en.json:440]: Unused

**Review by manuelmeister — APPROVED** [review]

**Review by pmattmann — APPROVED** [review]

**Review by carlobeltrame — APPROVED** [review]

**Pull request merged**